src/path.rs

```diff
@@ -70,6 +70,7 @@ impl<'a, Ctx: ResolverContextLike<'a>> PathString for EvaluationContext<'a, Ctx>
       "args" => convert_value(ctx.arg(tail)?),
       "headers" => ctx.header(tail[0].as_ref()).map(|v| v.into()),
       "vars" => ctx.var(tail[0].as_ref()).map(|v| v.into()),
+      "env" => std::env::var(tail[0].as_ref()).ok().map(|v| v.into()),
```

Comment:
Access env via ctx. Which internally accesses it via server context. That will make it easier to test.

Reply:
done.